xml/ha_storage_protection.xml

```diff
@@ -655,6 +664,16 @@ Timeout (msgwait)  : 10
     <para> If the SBD device is not accessible, the daemon will fail to start and inhibit
      &corosync; start-up. </para>
    </step>
+   <step xml:id="st.ha.storage.protect.sbd_delay_start">
+    <para>Search for the following parameter: <parameter>SBD_DELAY_START</parameter>.</para>
+    <para>
+      Enables or disables a delay. Set it to <literal>yes</literal>, if you
+      want to delay the start of SBD on boot. This parameter can be
+      necessary if your cluster nodes boots very fast whereas the other
+      nodes are still waiting to return from the fencing action.
```

Comment:
Let's mention that it's related to "msgwait" here again.  How about something like "In the case where msgwait needs to be relatively long, but your cluster nodes boots very fast ..."?

Reply:
Good idea! I slightly rephrased your sentence and combined it with what we have:

```xml
Enables or disables a delay. Set <parameter>SBD_DELAY_START</parameter>
to <literal>yes</literal> if <literal>msgwait</literal> is relatively
long, but your cluster nodes boot very fast. 
Setting this parameter to <literal>yes</literal> delays the start of
SBD on boot.  This is sometimes neccessary with virtual machines.
```